book/getting-started.rst

```diff
@@ -0,0 +1,140 @@
+Getting Started
+===============
+
+Are you ready for a quick start with Sulu? Sit back, fasten your seat belts
+and... go!
+
+Bootstrap a Project
+-------------------
+
+We'll bootstrap a new project based on the `Sulu Minimal Edition`_ with
+Composer_:
+
+.. code-block:: bash
+
+    composer create-project sulu/sulu-minimal my-project -n
+
+This command will bootstrap a new project in the directory ``my-project``.
+
+.. tip::
+
+    Now is a good time to start versioning your project. If you use Git,
+    initialize a new Git repository and submit your first commit:
+
+    .. code-block:: bash
+
+        cd my-project
+        git init
+        git add .
+        git commit -m "Initial commit"
+
+Webspaces
+---------
+
+The content management part of Sulu is built upon *webspaces*. Each of these
+webspaces configure a content tree. Each content tree may contain translations
+for different locales.
+
+The default webspace configuration is located in
+``app/Resources/webspaces/example.com.xml``. Rename this file so that it matches
+the name of your project.
+
+To get started, you only need to modify the ``<key>`` in that file. Change it
+to the name of your project:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <webspace xmlns="http://schemas.sulu.io/webspace/webspace"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/webspace/webspace http://schemas.sulu.io/webspace/webspace-1.0.xsd">
+
+        <name>My Project</name>
+        <key>my-project</key>
+
+        <!-- ... -->
+    </webspace>
+
+.. Caution::
+
+    Changing the ``<key>`` of a webspace later on causes complications. We
+    recommend to decide what key to use before you build the database in the
+    next step.
+
+We'll :doc:`return to webspaces <webspaces>` later in this book.
+
+Setup the Database
+------------------
+
+Next we'll setup a database for Sulu. You can use Sulu with any of the
+database backends `supported by Doctrine DBAL`_.
+
+Once you created a database, user and password, adapt the ``database_*``
+keys of your ``app/config/parameters.yml`` file. Here is an example for using
+Sulu with MySQL:
+
+.. code:: yaml
+
+    parameters:
+        database_driver: pdo_mysql
+        database_host: 127.0.0.1
+        database_port: null
+        database_name: hellosulu
+        database_user: hellosulu
+        database_password: averystrongpassword
+        database_version: 5.6
+
+.. tip::
+
+    The :doc:`parameter reference <../reference/parameters>` contains more
+    information about each of the parameters in this file.
+
+When you're done with the configuration, populate the database with Sulu's
+default data:
+
+.. code-block:: bash
+
+    bin/adminconsole sulu:build dev
```

Comment:
Shouldn't we explain here that a standard user is created with that command? Wouldn't be ideal if every Sulu installation has a `admin/admin` user 😃